ansible/roles/debops.postldap/defaults/main.yml

```diff
@@ -0,0 +1,627 @@
+# .. envvar:: postldap__ldap_self_attributes [[[
+#
+# YAML dictionary that defines the attributes of the LDAP object used by the
+# Postfix service to access the LDAP directory.
+postldap__ldap_self_attributes:
+  uid: '{{ postldap__ldap_self_rdn.split("=")[1] }}'
+  userPassword: '{{ postldap__ldap_bindpw }}'
+  host: '{{ [ ansible_fqdn, ansible_hostname ] | unique }}'
+  description: 'Account used by the "Postfix" service to access the LDAP directory'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_binddn [[[
+#
+# The Distinguished Name of the account LDAP object used by the
+# Postfix service to bind to the LDAP directory.
+postldap__ldap_binddn: '{{ ([ postldap__ldap_self_rdn ]
+                                        + postldap__ldap_device_dn) | join(",") }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_bindpw [[[
+#
+# The password stored in the account LDAP object used by the Postfix service
+# to bind to the LDAP directory.
+postldap__ldap_bindpw: '{{ lookup("password", secret + "/ldap/credentials/"
+                                  + postldap__ldap_binddn | to_uuid + ".password length=32 "
+                                  + "chars=alpha,digits,!@_#$%^&*") }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_people_rdn [[[
+#
+# The Relative Distinguished Name of the LDAP object which contains the user
+# accounts stored in LDAP.
+postldap__ldap_people_rdn: '{{ ansible_local.ldap.people_rdn
+                               if (ansible_local|d() and ansible_local.ldap|d() and
+                                   ansible_local.ldap.people_rdn|d())
+                               else "ou=People" }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_people_dn [[[
+#
+# The Distinguished Name of the LDAP object which contains the user accounts
+# used by Postfix.
+postldap__ldap_people_dn: '{{ [ postldap__ldap_people_rdn ]
+                                          + postldap__ldap_base_dn }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_uri [[[
+#
+# List of LDAP URIs that point to the directory servers which should be used by
+# `postfix virtual`.
+postldap__ldap_uri: '{{ ansible_local.ldap.uri
+                                    if (ansible_local|d() and ansible_local.ldap|d() and
+                                        ansible_local.ldap.uri|d())
+                                    else [""] }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_private_subtree [[[
+#
+# When this variable is enabled, the :ref:`debops.ldap` role will create
+# separate LDAP objects that manage the Postfix groups as subtree of the
+# DokiWiki service LDAP object. If you set this parameter to ``False``, the
```

Comment:
You should change this to "Postfix service LDAP object". It's a different role, after all. :-)